forms-flow-api/tests/unit/api/test_draft.py

```diff
@@ -129,11 +129,6 @@ def test_draft_submission_resource(app, client, session, jwt):
     assert response.json.get("formId") == payload["formId"]
     assert response.json.get("submissionId") == payload["submissionId"]
 
-    draft = Draft().query.get(draft_id)
-    assert draft.status == "0"
-    assert draft.data == {}
-    assert draft.application_id == response.json.get("id")
-
```

Comment:
I would comment out instead of removing them to make the future upgrade and merge conflict easier.

Reply:
Commented code